apps/book/forms.py

```diff
@@ -2,18 +2,32 @@
 from django import forms
 from django.utils.translation import ugettext_lazy as _
 from tinymce.widgets import TinyMCE
-from apps.book.models import Book
+from apps.book.models import Book, Author
 
 
 class BookAdminForm(forms.ModelForm):
     """
     Book admin form
     """
-    description_en = forms.CharField(widget=TinyMCE(attrs={'cols': 80, 'rows': 30}), label=_("Description"))
+    description_en = forms.CharField(widget=TinyMCE(attrs={'cols': 80, 'rows': 30}), label=_("Description [en]"))
     description_ne = forms.CharField(
         widget=TinyMCE(attrs={'cols': 80, 'rows': 30}), label=_("Description [ne]"), required=False
     )
 
     class Meta:
         model = Book
         fields = '__all__'
+
+
+class AuthorAdminForm(forms.ModelForm):
+    """
+    Author admin form
+    """
+    about_author_en = forms.CharField(widget=TinyMCE(attrs={'cols': 80, 'rows': 30}), label=_("About Author [en]"))
+    about_author_ne = forms.CharField(
+        widget=TinyMCE(attrs={'cols': 80, 'rows': 30}), label=_("About author [ne]"), required=False
+    )
```

Comment:
Can you check if this can be generated through ModelTranslation directly? We may need to repeat this if we need to add a language.

Reply:
@thenav56  We are using tinymce editor, so we have to create a form. This overrides field label generated form model translation, that's why we have to define fields label in form. (If we don't use tinymce editor then no need to define form too)